pkg/api/login_oauth.go

```diff
@@ -190,6 +190,23 @@ func (hs *HTTPServer) OAuthLogin(ctx *m.ReqContext) {
 		}
 	}
 
+	orgToRoleMap := connect.OrgToRoleMap()
+
+	for _, group := range userInfo.Groups {
+		if orgs, ok := orgToRoleMap[group]; ok {
+			for _, org := range orgs {
+				if org.OrgID == 0 {
```

Comment:
What is the idea of this check? in what case OrgID is 0?

Reply:
When OrgId is omitted from the toml configuration files.

Reply:
Maybe add a comment for clarity.